### DISCUSSION TITLE
Crashed simulations of EI-MS of LSD

### DISCUSSION THREAD

**Discussion by ghost** [qcxms/QCxMS#65] [Q&A] [answered]
I tried out 4 EI-MS simulations of the same substance (LSD) with different parameters:
Sim 1: `xtb, tmax 750, eimp0 80, ieeatm 0.9`
Sim 2: `xtb, tmax 750`
Sim 3: `xtb,  tmax 10` - crashed
Sim 4: `xtb, tmax 10, eimp0 80, ieeatm 0.9` - crashed.
The input structure is 

```
49

C          3.32715        3.41888       -0.03308
C          4.72656        3.78590        3.35568
C          7.03203       -1.11069       -3.44403
C          3.58976        2.32444        0.99448
C          5.58041        2.83748        2.53116
C          3.98793       -4.77411        2.28661
C          4.29736       -3.50945        1.76451
C          4.07258       -5.93820        1.50634
C          4.80931       -0.90771        0.20861
C          5.44396       -3.47091       -2.50843
C          5.06389       -6.02348       -1.99498
C          6.48791        0.05272       -1.40524
C          5.11306       -4.67970       -1.70243
C          5.34977        0.36526       -0.41041
C          4.69608       -3.36230        0.42932
C          5.11800       -2.12758       -0.27530
C          4.47168       -5.77256        0.17424
C          5.96206       -2.30648       -1.57988
C          4.76156       -4.52877       -0.34625
C          5.88474        1.26396        0.71924
N          4.66974       -6.67939       -0.84202
N          6.07871       -1.02041       -2.33880
N          5.00554        2.15722        1.36000
O          7.05247        1.15058        1.10857
H          2.27447        3.38667       -0.34115
H          3.93792        3.29523       -0.93303
H          3.52129        4.41480        0.37542
H          5.33067        4.18344        4.18138
H          3.87009        3.27555        3.80549
H          4.36997        4.63442        2.76718
H          7.11371       -0.14498       -3.95649
H          6.68831       -1.81574       -4.20723
H          8.03145       -1.41028       -3.11133
H          3.21518        1.38603        0.58179
H          2.99615        2.49864        1.89537
H          5.93302        2.04432        3.20410
H          6.47516        3.37547        2.19340
H          3.68890       -4.85673        3.32878
H          4.24525       -2.64744        2.42216
H          3.84484       -6.91238        1.92385
H          4.18770       -0.79444        1.09078
H          4.54390       -3.14888       -3.04545
H          6.20541       -3.73612       -3.25044
H          5.26469       -6.57456       -2.90009
H          7.42039       -0.22788       -0.89417
H          6.70729        0.97480       -1.95891
H          4.55498        0.87412       -0.96197
H          6.96803       -2.59820       -1.23630
H          4.55359       -7.68050       -0.75815
```
I think the last 2 simulations crashed because the simulation time is shorter than the time scale of the internal dynamics, but I'm not sure. The first run of `qcxms` for the generation of ground state trajectories was fine. The second run of `qcmxs` was when the program got killed. Unfortunately, there were no instructive error messages.

**Comment by JayTheDog** [comment, marked as answer]
`tmax` is the time for the sampling of the MDs.  So it depends a bit when you reduced the value - directly at the beginning or after the groundstate sampling?
Reducing the `tmax` to a too small value for a large or flexible molecule will try to access structures for which no snapshot is available.
That means increasing the `tmax` by some value will fix this. Maybe try `50`, and increase if it still crashes, or reduce if you want to reduce the total timing, until you see crashing again.

fyi, as soon as you have sampled the groundstate MD, you do not need to re-run the GS again for each run. i.e. you can use the `.gs` file as starting point for a production run at e.g. `eimp 80` and `tmax 10`, but do not need to re-run the "first" `qcxms` call. 
**Note**: Using a "longer" GS MD for shorter production run simulations is no problem (up to a certain value), while running longer production runs with a short GS MD **won't work**!
